controller/schema.go

```diff
@@ -271,6 +271,24 @@ $$ LANGUAGE plpgsql`,
 		$$`,
 		`UPDATE apps SET meta = jsonb_merge(meta, '{"flynn-system-critical":"true"}') WHERE name IN ('discoverd', 'flannel', 'postgres', 'controller')`,
 	)
+	migrations.Add(14,
+		`CREATE TABLE backup_statuses (name text PRIMARY KEY)`,
+		`INSERT INTO backup_statuses (name) VALUES
+			('running'), ('complete'), ('error')`,
+		`CREATE TABLE backups (
+			backup_id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+			status text NOT NULL,
+			sha512 bytea,
+			size integer,
+			error text,
+			created_at timestamptz NOT NULL DEFAULT now(),
+			updated_at timestamptz NOT NULL DEFAULT now(),
+			completed_at timestamptz,
+			deleted_at timestamptz
+		)`,
+		`ALTER TABLE backups ADD CONSTRAINT backups_status_fkey FOREIGN KEY (status) REFERENCES backup_statuses (name)`,
```

Comment:
This can be dropped and you should be able to put `REFERENCES backup_statuses (status)` after `status text NOT NULL` instead.